### ISSUE TITLE
Old OSM data

### ISSUE THREAD

**Issue by BigKananga** [Qwant/qwantmaps#193]
The OSM data haven't been updated at least since February. Because there are many changes (e.g. POIs) and in former times data have been updated every few weeks, I'm wondering when the next data update is planned and how often they will be updated in future?

**Comment by jaysonwcs** [comment]
I've updated a lot of info on the town I live in a month ago, and I noticed the outdated data too. I'm curious about this info too.

**Comment by sdrll** [comment]
Hi, we have a technical issue for the update. Last month's lack of prioritization for maps. It should be fixed in the next weeks.

**Comment by BigKananga** [comment]
@sdrll 
Thanks for information!

**Comment by BigKananga** [comment]
@sdrll 
Any news? Two months later still no data update.

**Comment by sdrll** [comment]
Hi, we have 2/3 of our environment that are updated. We are waiting the last 1/3 to deploy the updated OSM content to everyone. (we are facing an infrastructure migration that impact multiples things)

**Comment by n-peugnet** [comment]
Hi, it seems that the data is still oudated of multiple months. At least in places I know. Are you still struggling with the infrastructure migration ?

**Comment by BigKananga** [comment]
Yes, data are still from February.

**Comment by sdrll** [comment]
Hi, yes basically we have migrated from Docker Swarm to Kubernetes in our on premise infrastructure. (Very small steps are needed to put it on production). But, because Qwant was taken over, we have to migrate to OVH Cloud now. I am waiting that everyone return from holidays for taking the next actions. Sorry for the delay

**Comment by sdrll** [comment]
Hi, should be updated during next week. After that, it should be updated every week.

**Comment by sdrll** [comment]
It has finally been released. Sorry for the long wait.

**Comment by sdrll** [comment]
we just had an issue, we will re-release tomorrow

**Comment by jmarchon1** [comment]
Still appears to be out of date to me, at least in the Pacific Northwest. Would love to see an update sometime soon.

**Comment by BigKananga** [comment]
> It's a lot better, but from what I'm seeing based on my past edits it caught up to OSM data of ~4 months ago.

I can confirm this. Saw several ways/buildings and nodes where edits one or two months ago are not included. But it is better than the old version from February of course.

**Comment by shibotto** [comment]
> Do you have an example, please ?

Here is a node showing correctly (2023-05-31):
https://www.openstreetmap.org/node/10942893803
https://www.qwant.com/maps/place/osm:node:10942893803

Here is one not showing (2023-06-30):
https://www.openstreetmap.org/node/2718188291
https://www.qwant.com/maps/place/osm:node:2718188291 (made up not working link, [the location](https://www.qwant.com/maps/#map=20.00/44.0397723/12.6018457) more or less)

**Comment by sdrll** [comment]
Indeed, I will update with the latest version. Thank you very much for your help

**Comment by sdrll** [comment]
Done, it should be up-to-date

**Comment by BigKananga** [comment]
Much more current now, but 2 things:

1. opening hours are not shown anymore
2. many useless "POIs" like small parks without names or private gardens
